functions/imagemagick/index.js

```diff
@@ -29,12 +30,12 @@ const {BLURRED_BUCKET_NAME} = process.env;
 
 // [START functions_imagemagick_analyze]
 // Blurs uploaded images that are flagged as Adult or Violence.
-exports.blurOffensiveImages = async event => {
+functions.cloudEvent('blurOffensiveImages', async cloudEvent => {
   // This event represents the triggering Cloud Storage object.
-  const object = event;
-
-  const file = storage.bucket(object.bucket).file(object.name);
-  const filePath = `gs://${object.bucket}/${object.name}`;
+  const bucket = cloudEvent.data.bucket;
```

Comment:
nit, we may want to check that data, bucket and name exist

Reply:
I'd argue this is distracting to users **if** we presume events are **a)** created programmatically and **b)** only come from `GCS -> Eventarc` to this function.

(In other words, we shouldn't have any _event-structure-related human errors_ outside of testing scenarios.)

Reply:
Resolved offline.